pkg/collector/cluster.go

```diff
@@ -127,3 +132,14 @@ func (c *ClusterCollector) Get() ([]map[string]interface{}, error) {
 
 	return results, nil
 }
+
+func (c *ClusterCollector) getLastAppliedConfig(r unstructured.Unstructured) (string, bool) {
+	annotations := append(c.additionalAnnotations, "kubectl.kubernetes.io/last-applied-configuration")
```

Comment:
I think that contrary to the PR description, the additional annotation(s) will have priority over the `last-applied...` one, as we return with the first one found. I don't have a strong case for either, but as mentioned - for compatibility - I would prefer the other way around.
```suggestion
	annotations := append([]string{"kubectl.kubernetes.io/last-applied-configuration"}, c.additionalAnnotations...)
```

Reply:
Yeah, you are right. Good catch, thanks! :slightly_smiling_face: